### ISSUE TITLE
Upfront text about how to cite BOUT++

### ISSUE THREAD

**Issue by ZedThree** [boutproject/boutproject.github.io#40]
We should have something on the homepage about how to cite BOUT++, preferably with something people can just copy+paste

**Comment by bendudson** [comment]
Agreed. There are several possible papers; we really need a more recent review paper than https://doi.org/10.1017/S0022377814000816 which has some authors but is notably missing @ZedThree, @JosephThomasParker and @d7919 which I think rules it out. With the latest release and developments in OpenMP and vectorisation, perhaps some kind of software paper could be submitted?

**Comment by ZedThree** [comment]
I am at a hackday for the Citation File Format, which might provide a solution. It can include additional papers that should be cited as well.

**Comment by dschwoerer** [comment]
Any chance to get this on the website?
Would be nice if I could just copy paste this :-)

Where should this go? Documentation? Own section?

**Comment by johnomotani** [comment]
Bump. A list of the Zenodo DOIs would be nice to have too :-)
